# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1377006532 14400
#      Tue Aug 20 09:48:52 2013 -0400
# Node ID 88a6f2f540adac25092eb4dab3ace9eb9b39cf7a
# Parent  011e5d67baf1a2caff4bb51bc8d9f1f12930f4cc
io.tst: Condition load-save test on HAVE_ZLIB

* test/io.tst: Condition load-save test on HAVE_ZLIB since testls tests
file formats that require zlib compression.

diff --git a/test/io.tst b/test/io.tst
--- a/test/io.tst
+++ b/test/io.tst
@@ -173,17 +173,17 @@
 %!        error ("failed: %s global test", file);
 %!      endif
 %!    endfor
 %!  endif
 %!
 %!  ret = 1;
 %!endfunction
 
-%!test
+%!testif HAVE_ZLIB
 %!
 %! [save_status, save_files] = testls (0);
 %! [load_status, load_files] = testls (1);
 %!
 %! for f = [save_files, load_files]
 %!   unlink (f{1});
 %! endfor
 %!
